integration/log.go

```diff
@@ -504,11 +500,10 @@ func checkConsistencyProof(consistParams consistencyProofParams, treeID int64, t
 }
 
 // buildMerkleTree returns an in-memory Merkle tree built on the given leaves.
-func buildMerkleTree(leafMap map[int64]*trillian.LogLeaf, params TestParameters) *inmemory.Tree {
+func buildMerkleTree(leaves []*trillian.LogLeaf, params TestParameters) *inmemory.Tree {
```

Comment:
This is why `StartLeaf` that is not 0 was not supported: note that this function builds an in-memory Merkle tree that starts from empty.